boards/xtensa/intel_s1000_crb/doc/index.rst

```diff
@@ -125,6 +125,10 @@ software keys you have downloaded from `Tensilica Tools for Sue Creek`_
    follow what the license manager expects or create a dedicated ethernet device
    for this to work.
 
+   On the latest LTS ubuntu(18.04), you probably have to install the license key
+   manually with "cp sue-creek-SDK-license.dat <path to  SDK>/XtDevTools/install/
```

Comment:
please put the command on a new line as a code-block.

Reply:
Hi Anas, I addressed your comment with commit  7853cf1. Please help to take look at it.